src/Ryujinx.HLE/HOS/Services/Hid/IHidbusServer.cs

```diff
@@ -1,8 +1,26 @@
-﻿namespace Ryujinx.HLE.HOS.Services.Hid
+﻿using Ryujinx.Common.Logging;
+
+namespace Ryujinx.HLE.HOS.Services.Hid
 {
     [Service("hidbus")]
     class IHidbusServer : IpcService
     {
         public IHidbusServer(ServiceCtx context) { }
+
+        [CommandCmif(1)]
+        // GetBusHandle(nn::hid::NpadIdType, nn::hidbus::BusType, nn::applet::AppletResourceUserId) -> bool HasHandle, nn::hidbus::BusHandle
```

Comment:
```suggestion
        // GetBusHandle(nn::hid::NpadIdType, nn::hidbus::BusType, nn::applet::AppletResourceUserId) -> (bool HasHandle, nn::hidbus::BusHandle)
```
IIRC that's how we do multiple value returns in the other comments.
Also it seems that you write the bool, but you never write the output `BusHandle`.

Reply:
The bool indicates whether the following structure is valid, since we write false the following data shouldn't matter. But I can add in some dummy structure data if you prefer.

Reply:
Updated to fix style and include handle data in the stubbed output.